### PR TITLE
[image-picker][Android][plugin] Migrate to `com.github.CanHub:Android-Image-Cropper`

### PR DESCRIPTION
**Pull request by bbarthec**
# Why

Part of [ENG-1858](https://linear.app/expo/issue/ENG-2858/remove-jcenter)

# How

Migrated to `https://github.com/CanHub/Android-Image-Cropper@1.1.1`.
Version 1.0.0 is published invalidly to the jitpack.io repository. 

# Test Plan

- [x] `bare-expo`#`image-picker-screen` - editing files works & `test-suite` works
- [x] `Expo Go@unversioned & @SDK44` works

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [x] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/expo-image-picker/android/build.gradle [hunk 1]
```diff
@@ -13,6 +13,7 @@ buildscript {
 
   repositories {
     mavenCentral()
+    maven { url 'https://jitpack.io' }
```
Comment (Kudo) [inline packages/expo-image-picker/android/build.gradle:16]: that's `buildscript.repositories` which may not what you want?
for app to download the dependency, we already have jitpack in our template:
https://github.com/expo/expo/blob/594bb7e16883bf345e4bf14c3e6380e7df176c79/templates/expo-template-bare-minimum/android/build.gradle#L39

**Review by Kudo — APPROVED** [review]
other than the jitpack `buildscript.repositories`, this pr looks promising. thanks for the awesome work!

**Review by lukmccall — APPROVED** [review]

File: packages/expo-image-picker/android/build.gradle (hunk 1 above)
Comment (bbarthec) [reply packages/expo-image-picker/android/build.gradle:16]: Oh, true 🤦 
What would happen if somebody is not having `jitpack.io` in their main gradle file?
Isn't is better for us to have each package be self-describing and storing it's own independent configuration? 🤔 I follow the idea of `package`-centric approach where each package-related aspect is pushed down to the package itself, just to enable it's usage standalone-ly.

Comment (bbarthec) [inline packages/expo-image-picker/android/build.gradle:16]:
```suggestion
```

**Pull request merged**